app/styles/vendor/material-theme.scss

```diff
@@ -128,6 +128,10 @@ md-dialog-container.mat-dialog-container {
     width: auto !important;
     height: auto !important;
     max-height: 80vh;
+
+    form {
```

Comment:
hhm don't like this, why on all forms?

Reply:
its not on all forms, only on the "mat-dialog-container" as the padding that was on here has been removed and the dialog's look funny with no padding. ill add a before and after pic.

Reply:
before 

![no-padding](https://user-images.githubusercontent.com/10934358/30007126-e3e0ecb4-915b-11e7-9bec-b47a62da2bf1.PNG)

after

![padding](https://user-images.githubusercontent.com/10934358/30007128-e7d8080c-915b-11e7-8716-73d9ce76b178.PNG)

Reply:
I think this was because I try to remove some padding with the simple form but I reverted that in the upload pr